base/db/R/query_pfts.R

```diff
@@ -0,0 +1,40 @@
+#' Retrieve PFT ID, name, and type from BETY
+#'
+#' @param dbcon Database connection object
+#' @param pft_names Character vector of PFT names
+#' @param modeltype 
+#' @return `data.frame` containing PFT ID (`id`), type (`pft_type`),
+#'   and name (`name`). 
+#' @author Alexey Shiklomanov, Chris Black
+#' @export
+query_pfts <- function(dbcon, pft_names, modeltype = NULL, strict = FALSE) {
+  pft_names <- pft_names # Force evaluation here to avoid dbplyr issues
```

Comment:
I've only seen it in other people's code and never used it myself, so... 

is this 🦋 a use case for `force()`?

Reply:
Maybe? I have no idea how `force` relates to `dbplyr`'s implicit SQL translation, but I think you may be right since I think I've seen somewhere (Hadley's Advanced R) that `force` is exactly the same as just evaluating the object.

Reply:
Turns out this doesn't actually need to be there -- the function works as expected even without it. I guess R's standard evaluation of function calls supercedes tidy evaluation? 🤷‍♂